subsys/bluetooth/audio/Kconfig.cap

```diff
@@ -15,10 +15,13 @@ config BT_CAP_ACCEPTOR
 	depends on BT_AUDIO_UNICAST_SERVER || (BT_AUDIO_BROADCAST_SINK && BT_BASS)
 	select EXPERIMENTAL
 	select BT_CAP
+	help
+	  Enabling this will enable the CAP Acceptor role. This instantiates the
+	  common audio service (CAS).
 
-config BT_DEBUG_CAP_ACCEPTOR
+config BT_DEBUG_CAP
 	bool "Common Audio Profile Acceptor debug"
-	depends on BT_CAP_ACCEPTOR
+	depends on BT_CAP
 	help
-	  Use this option to enable CAP Acceptor debug logs for the
+	  Use this option to enable CAP debug logs for the
```

Comment:
These changes to the Kconfig.cap (2nd commit - ASCS metadata) could be merged with the first commit.
They do not really seem relevant in this commit.